### ISSUE TITLE
System Error when running Ganache 1.1.0 on linux

### ISSUE THREAD

**Issue by ccolorado** [trufflesuite/ganache#911]
Ganache cannot launch without error anymore, from either console or the application runner 

Tried setting ether balance on ganache. As suggested in this post for ganache-cli.
https://ethereum.stackexchange.com/questions/38738/setting-the-initial-balance-of-an-account-in-ganache-cli

Ganache launched with an error
ganache --help ( on the console ) launches the application which shouldn't  ( so does ganache --version)
ganache can only launch with an error now. 

The application was launched with the attached error message.

PLATFORM: linux
GANACHE VERSION: 1.1.0

EXCEPTION:
```
Error: listen EADDRINUSE 127.0.0.1:7545
    at Object.exports._errnoException (util.js:1024:11)
    at exports._exceptionWithHostPort (util.js:1047:20)
    at Server.setupListenHandle [as _listen2] (net.js:1319:14)
    at listenInCluster (net.js:1367:12)
    at doListen (net.js:1493:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: Starting server with initial configuration: {"hostname":"127.0.0.1","port":7545,"network_id":5777,"total_accounts":10,"unlocked_accounts":[],"vmErrorsOnRPCResponse":true,"gasLimit":22223000,"gasPrice":4100000000}
```

**Comment by ccolorado** [comment]
Nevermind had a ganache instance running on the background preventing ganache to launch.

**Comment by seesemichaelj** [comment]
Hey @ccolorado!

As you figured out, 1.1.0 has issues starting up if there is something else bound to the same port. However, there is also a bug where Ganache UI will keep the port bound when it should close/etc, running into the same error.

The issue #665 was resolved in 1.2.x, and we also made it so that you can change the port easily if there is something else running.

I highly recommend upgrading Ganache! Go get the latest release here: https://github.com/trufflesuite/ganache/releases/latest
